libs/isograph-react/src/core/cache.ts

```diff
@@ -730,13 +730,35 @@ export function getParentRecordKey(
 function getStoreKeyChunkForArgumentValue(
   argumentValue: ArgumentValue,
   variables: Variables,
-) {
+): string {
   switch (argumentValue.kind) {
+    case 'Object': {
+      return JSON.stringify(
+        stableCopy(
+          Object.fromEntries(
+            argumentValue.value.map(([argumentName, argumentValue]) => {
+              return [
+                argumentName,
+                //  substitute variables
+                getStoreKeyChunkForArgumentValue(argumentValue, variables),
```

Comment:
Just thinking out loud... Damn... so we have to serialize large objects whenever we read a field that takes a large object. I remember that this was a big chunk of the reading time in Relay. I wonder if there's stuff we can do.